test/integration/fixtures/options/only/bdd.fixture.js

```diff
@@ -47,26 +47,43 @@ describe.only('should run this suite, even', function () {
 describe('should run this suite with an exclusive test', function () {
   it.only('should run this test', function () {});
 
-  describe('should not run this nested suite', function () {
-    describe.only('should not run this doubly-nested suite', function () {
-      it('should not run this test', function () {});
+  it('should not run this test', function () {
+    (true).should.equal(false);
+  });
+
+  describe('should run this nested suite', function () {
+    it('should not run this test', function () {
+      (true).should.equal(false);
+    });
 
-      it('should not run this test', function () {});
+    describe.only('should run this doubly-nested suite', function () {
+      it('should run this test', function () {});
 
-      it('should not run this test', function () {});
+      it('should run this test', function () {});
+
+      it('should run this test', function () {});
     });
   });
 });
 
 describe('should run this suite with an exclusive test (reverse order)', function () {
-  describe('should not run this nested suite', function () {
-    describe.only('should not run this doubly-nested suite', function () {
-      it('should not run this test', function () {});
+  describe('should run this nested suite with an exclusive suite', function () {
+    describe.only('should run this doubly-nested suite', function () {
+      it('should run this test', function () {});
+
+      it('should run this test', function () {});
 
-      it('should not run this test', function () {});
+      it('should run this test', function () {});
+    });
 
-      it('should not run this test', function () {});
+    it('should not run this test', function () {
+      (true).should.equal(false);
     });
   });
+
   it.only('should run this test', function () {});
+
+  it('should not run this test', function () {
+    (true).should.equal(false);
+  });
 });
```

Comment:
Adding failing assertions to the should-not-run tests here is misleading, since what matters is the integration test verifying that they did not run (and that the should-run tests did). It could, however, be appropriate to add a comment explaining that setup instead, if you'd like to help make it more obvious for future contributors.